data/webshells.txt

```diff
@@ -5,4 +5,5 @@ eval\(base64_decode\(
 "base" \. "64_decode"
 ; for\(\$i=0; \$i < strlen\(
 \?php \${
-=\$GLOBALS;\${
\ No newline at end of file
+=\$GLOBALS;\${
+\$GLOBALS\[\$GLOBALS\['
```

Comment:
I don't think this is part of the preg_line fix.

What is it? This should be on a separated commit.